tests/Integration/Behaviour/Features/Scenario/Discount/FO/full_user_experience_test_with_discount_cart_level.feature

```diff
@@ -0,0 +1,80 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s discount --tags full-ux-discount-test-cart-level
+@restore-all-tables-before-feature
+@full-ux-discount-test-cart-level
+
+Feature: Full UX discount test
+  PrestaShop allows BO users to create discounts
+  As a BO user
+  I must be able to create discounts using the new discounts
+
+  Background:
+    Given I have an empty default cart
+    Given shop "shop1" with name "test_shop" exists
+    And there is a currency named "usd" with iso code "USD" and exchange rate of 0.92
+    And shop configuration for "PS_CART_RULE_FEATURE_ACTIVE" is set to 1
+    And there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
+    And language with iso code "en" is the default one
+
+  Scenario: Create a complete cart level flat discount using new CQRS
+    When I create a cart level discount "complete_amount_cart_level_discount" with following properties:
+      | type               | amount_discount        |
+      | name[en-US]        | Promotion              |
+      | active             | true                   |
+      | valid_from         | 2025-01-01 11:05:00    |
+      | valid_to           | 2025-12-01 00:00:00    |
+      | free_shipping      | false                  |
+      | code               | PROMO_CART_2025        |
+      | reduction_amount   | 10.0                   |
+      | reduction_currency | usd                    |
+      | taxIncluded        | true                   |
+    And discount "complete_amount_cart_level_discount" should have the following properties:
+      | name[en-US]        | Promotion              |
+      | active             | true                   |
+      | valid_from         | 2025-01-01 11:05:00    |
+      | valid_to           | 2025-12-01 00:00:00    |
+      | free_shipping      | false                  |
+      | code               | PROMO_CART_2025        |
+      | reduction_amount   | 10.0                   |
+      | reduction_currency | usd                    |
+      | taxIncluded        | true                   |
+
+  Scenario: One product in cart, one amount cart level discount
+    Given discount "complete_amount_cart_level_discount" should have the following properties:
+      | name[en-US]   | Promotion       |
+      | free_shipping | true            |
```

Comment:
I don't understand how `free_shipping` turns into true all of a sudden when it was false in the previous step
And behat tests are still green besides this illogical behaviour, are you sure this field is tested?

Reply:
I just checked to be sure, seems like there is no assertion regarding `free_shipping` in the Context class
If it's not relevant because it's not part of the returned data in the CQRS query it's fine, but then no need to have those lines in the feature files, they should be removed everywhere